src/collectd-python.pod

```diff
@@ -475,7 +475,7 @@ Methods defined here:
 
 =over 4
 
-=item B<dispatch>([type][, values][, plugin_instance][, type_instance][, plugin][, host][, time][, interval]) -> None.  Dispatch a value list.
+=item B<dispatch>([type][, message][, plugin_instance][, type_instance][, plugin][, host][, time][, severity][, meta]) -> None.  Dispatch a notification.
```

Comment:
Thanks for noticing this! I've cherry-picked this fix into *collectd-5.7* as be906f960ede42f72317eab6aa6a5f116857a484.